functional/testHeadlessComponents/playlist.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TKG/resources/playlist.xsd">
+	<test>
+		<testCaseName>THC</testCaseName>
```

Comment:
Prefer a testCaseName that was more descriptive, there is no value in acronyms or shortnames here, please update to TestHeadlessComponents or something that indicates what the test does.

Reply:
Sure sorry.. I figured that since I will write it like 20 times into grinder I may as well make it easier for myself, but will change :-)